src/components/Home/Hero/GetStarted/GetStartedWithDvcX.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import HeroContainer from '../../../HeroContainer'
+import { StaticImage } from 'gatsby-plugin-image'
+import { cn } from '../../../../utils'
+import DvcxSlides from '../../LandingHero/DvcxSlides'
+import { ReactComponent as ArrowRight } from '../../../../../static/img/arrow_right_white.svg'
+
+import { CTAButton } from '../HeroSection'
+
+const GetStartedWithDvcX = () => {
+  return (
+    <HeroContainer className="py-10 px-6">
+      <div className="flex items-center md:justify-center">
+        <h1 className={cn('text-2xl font-medium')}>Get Started with</h1>
+        <StaticImage
+          height={40}
+          src="../../../../../static/img/logos/dvcx.svg"
+          alt="DVCx Logo"
+        />
+      </div>
+      <DvcxSlides />
```

Comment:
Can you reverse the order of the terminal and accordion for DVCX? Accordion on the right, terminal on the left.

This will help increase the visual separation between the two "Get Started" sections and help people understand how this section aligns with the user journey diagram from the hero above.